src/Symfony/Component/DomCrawler/Test/Constraint/CrawlerSelectorTextContains.php

```diff
@@ -56,7 +56,7 @@ protected function matches($crawler): bool
         $this->hasNode = true;
         $this->nodeText = $crawler->text(null, true);
 
-        return false !== mb_strpos($this->nodeText, $this->expectedText);
+        return str_contains($this->nodeText, $this->expectedText);
```

Comment:
From the str_contains RFC:
> In a recent discussion in the internals mailing list, we came to the conclusion, there is no need for a multibyte variant of this function (e.g. mb_str_contains). The reason behind this is: A multibyte variant of this function would behave no different than the non-multibyte function. Multibyte variants behave differently when the offset/position has relevance at which the string was found. As this is not the case for this function, there is no need for that.

And from what I understood, it's the same for `str_starts_with()` and `str_ends_with()`.